stellar/contract.js

```diff
@@ -4,28 +4,38 @@ const { Contract, SorobanRpc } = require('@stellar/stellar-sdk');
 const { Command, Option } = require('commander');
 const { execSync } = require('child_process');
 const { loadConfig, printInfo, saveConfig } = require('../evm/utils');
-const { stellarCmd, getNetworkPassphrase, addBaseOptions } = require('./utils');
+const { stellarCmd, getNetworkPassphrase, addBaseOptions, getWallet, broadcast } = require('./utils');
 const { getChainConfig, addOptionsToCommands } = require('../common');
 const { prompt } = require('../common/utils');
 require('./cli-utils');
 
 const MAX_INSTANCE_TTL_EXTENSION = 535679;
 
-async function getTtl(chain, contractName, _args, _options) {
+async function handlePauseOperation(chain, _, contract, arg, options) {
```

Comment:
```suggestion
async function submitOperation(chain, _, contract, operation, args, options) {
```
Could keep it generic